kong/templates/kong_defaults.lua

```diff
@@ -43,6 +43,7 @@ cluster_use_proxy = off
 cluster_dp_labels = NONE
 cluster_rpc = off
 cluster_incremental_sync = off
+cluster_full_sync_threshold = 512
```

Comment:
Shouldn't a description of this configuration item be added to the `kong.conf.default` file as well?

Reply:
Currently other items of incremental sync are not in kong.conf.default, like `cluster_rpc `, so it did the same thing.

Reply:
so i think we could resolve this comment, when GA, we will add these options~